sklearn/utils/tests/test_estimator_checks.py

```diff
@@ -1422,7 +1422,12 @@ def test_xfail_count_with_no_fast_fail():
             expected_failed_checks=expected_failed_checks,
             on_fail="warn",
         )
-    xfail_warns = [w for w in records if w.category != SkipTestWarning]
+    xfail_warns = [
+        w
+        for w in records
+        if w.category != SkipTestWarning and w.category != FutureWarning
+    ]
```

Comment:
Rather than doing this, we should probably replace `est` by an instance of a class that does not raise any `FutureWarning` while running the common tests. `LogisticRegression` or `Ridge` might work for that purpose.